### PR TITLE
py-kaptan: new submission

### PR DESCRIPTION
**Pull request by egorenar**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.1 18B75
Xcode 10.1 10B61

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-colorama/Portfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-colorama
-version             0.3.7
+version             0.3.9
```
Comment (yan12125) [inline python/py-colorama/Portfile]: Don't mix changes from another port.

File: python/py-kaptan/Portfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-kaptan
+version             0.5.10
+maintainers         nomaintainer
```
Comment (yan12125) [inline python/py-kaptan/Portfile]: You should be a maintainer.

File: python/py-kaptan/Portfile [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+description         Configuration parser.
+long_description    ${description}
+
+platforms           darwin
+license             BSD
+
+homepage            https://github.com/emre/kaptan
+master_sites        pypi:k/${python.rootname}/
+distname            ${python.rootname}-${version}
+checksums           md5     d8c9eab27e915b9aae5b65fdb7b0e834 \
```
Comment (yan12125) [inline python/py-kaptan/Portfile]: md5 and sha1 checksums are not necessary - they have been proved to be not-so-secure, so using them is not useful.

File: python/py-kaptan/Portfile [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-kaptan
+version             0.5.10
+maintainers         {egorenar @egorenar} openmaintainer
```
Comment (yan12125) [inline python/py-kaptan/Portfile]: FYI: the first field `egorenar` indicates the email address egorenar@macports.org. Looks like you're not a MacPorts member yet? If so you should use your full email address.

See the "maintainers" section in https://guide.macports.org/#reference.keywords for details.

Comment (egorenar) [reply python/py-kaptan/Portfile]: thanks, fixed.

**Comment by yan12125** [comment]
Thanks for submitting this!

**Pull request merged**